### PR TITLE
Revert "Branch management: the 3 roads (#247)"

### LINKED ISSUE #284: Is the three branches system really worth it?
Since #153 was written, voted and applied on some project, some of the @sonata-project/contributors already complain about this system.

The principal reason? Really heavy to maintain. And I can understand that.

So this is a simple vote issue:

:+1: To going back to the 2 roads branches.
:-1: To stay on the 3 roads branches.

If we go back to the 2 roads classic system, be warn that the schema will be like (**starting the next major**):

- `master`: The unstable branch, like actual (5.x for core-bundle)
- `stable`: For BC features, deprecation and bug fixes (4.x for core-bundle)

And nothing more. The legacy branch must not be maintained anymore and will not be managed by dev-kit.

Thanks to use reaction emoji to vote.

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Back to the old days, brothers!

Closes #284 

This reverts commit 16cb431f77169629335142f8aff65e7b04093cb2.

### CODE REVIEW

File: config/projects.yml [hunk 1]
```diff
@@ -260,8 +260,6 @@ google-authenticator:
       php: ['7.1']
     2.x:
       php: ['7.0', '7.1']
-    1.x:
-      php: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1']
```
Comment (greg0ire) [inline config/projects.yml]: Uuuuh what?

Comment (Soullivaneuh) [reply config/projects.yml]: If we go back to the 2 branch road, we have to remove it.

Comment (greg0ire) [reply config/projects.yml]: Oh right it's a removal

**Comment by Soullivaneuh** [comment]
@greg0ire Could you please review my note about legacy branches?

File: project/CONTRIBUTING.md [hunk 2]
```diff
@@ -276,6 +271,13 @@ or postponed to next major release if BC is not possible.
 
 If you are not sure what should be done, don't hesitate to open an issue about your PR project.
 
+##### Legacy branches
+
+Legacy branches are **NOT** supported at all. Any submitted Pull Request will be immediately closed.
+
+Core teams member *may* cherry-pick some fixes from the stable branch to the legacy one if it's really needed
```
Comment (greg0ire) [inline project/CONTRIBUTING.md]: "team members"

**Pull request merged**